web/src/views/FindSpeaker.vue

```diff
@@ -134,7 +134,7 @@ export default {
       return lastPageEntry > this.speakers.length ? this.speakers.length : lastPageEntry;
     },
   },
-  mounted() {
+  async mounted() {
```

Comment:
Why this change?

Reply:
Oops, forgot to remove this. Good catch.